source/configuring_domain/02_managing_legal_information/legals.rst

```diff
@@ -64,9 +64,26 @@ To specify the Terms and Conditions and the Privacy Policy of a portal, go to Co
    Fill-in Terms & Conditions and Privacy Policy templates are available, based on the French law (in French and English languages) and the US law (in English language) only. Click the Use a template button under the textbox to use a template. The generated conditions of use can be edited in the textbox afterward.
 
 
+Licenses
+--------
+
+When configuring the metadata of a dataset in the Information tab, the license attached to that dataset should be defined, using a drop-down selection of predefined licenses. To define the licenses listed in the drop-down selection, go to Configuration > Legals. In this subsection of the back office menu, look for the "Licenses" area.
+
+.. image:: images/license_configuration.png
+
+1. Click the Add license button.
+2. In the Labels column, write the label of the license. The label will be displayed both in the drop-down selection in the back office, and in the Information tab of the published dataset in the front office. There must be one label per language available on the domain.
+3. (optional) In the URLs column, indicate the official website of the defined license, in order to find more information about that license. There can be one URL per language available on the domain.
```

Comment:
What happens if an URL is not defined for one language? Does it fallback to another one or does it just considers there is no URL for this license in this specific language?

Reply:
If the URL is not defined for one of the languages, the license label in the front office will not be clickable at all. I'm going to add a note to precise this, in case users have that same question!